.eslintrc.js

```diff
@@ -29,6 +29,15 @@ module.exports = {
   'rules': {
     'strict': 0,
 
+    // Forbid multiple statements in one line
+    'max-statements-per-line': ["error", { "max": 1 }],
+
+    // Allow for-of loops
+    'no-restricted-syntax': ['error', 'ForInStatement', 'LabeledStatement', 'WithStatement'],
+
+    // Allow return before else & redundant else statements
+    'no-else-return': 'off',
+
```

Comment:
can you also suggest those in https://github.com/adobe-rnd/new-nodejs-repository-template/blob/master/.eslintrc.js ?